Fix ignore on request path (#2146)
elastic/apm-agent-java: CHANGELOG.asciidoc

[[release-notes-1.27.0]]
==== 1.27.0 - YYYY/MM/DD

[float]

> ===== Breaking changes

Reviewer: ```suggestion
===== Potentially breaking changes
```

* `transaction_ignore_urls` now relies on full request URL path - {pull}2146[#2146]
** On a typical application server like Tomcat, deploying an `app.war` application to the non-ROOT context makes it accessible with `http://localhost:8080/app/`
** Ignoring the whole webapp through `/app/*` was not possible until now.

> ** We can now ignore a whole webapp through `/app/*`.

Reviewer: ```suggestion
```
Seems like a duplication

> ** Existing configuration needs to be updated to include the deployment context, thus for example `/static/*.js` used to

Reviewer: ```suggestion
** Existing configuration may need to be updated to include the deployment context, thus for example `/static/*.js` used to
```

> exclude known static files in all applications might be changed to `/app/static/*.js`.

Reviewer: ```suggestion
exclude known static files in all applications might be changed to `/app/static/*.js` or `*/static/*.js`.
```
